lib/rollbar/request_data_extractor.rb

```diff
@@ -28,7 +28,12 @@ def extract_request_data_from_rack(env)
       session = rollbar_filtered_params(sensitive_params, rollbar_request_session(rack_req))
       route_params = rollbar_filtered_params(sensitive_params, rollbar_route_params(env))
 
-      params = request_params.merge(get_params).merge(post_params).merge(raw_body_params)
+      params =
+        if !raw_body_params.empty?
```

Comment:
@elight I'd say we are getting the `post_params` and `raw_body_params` cause Rack doesn't parse JSON payload when you ask for `request.POST`. But you are probably right in the sense that we are doing something wrong. We'll look in detail to this.

But anyway, I think this logic shouldn't be here in this PR. What do you think?